_posts/en/newsletters/2021-11-17-newsletter.md

```diff
@@ -0,0 +1,154 @@
+  BitGo [announced][bitgo taproot blog] support for both sending from and
+  receiving to [taproot][topic taproot] outputs using their API. Taproot support
+  in the UI is planned for a future update.
+
+- **NthKey adds bech32m send capabilities:**
+  iOS signing service [NthKey][nthkey website] added support for taproot sends
+  in the [v1.0.4 release][nthkey v1.0.4].
+
+- **Ledger Live supports taproot:**
+  Ledger's client software, Ledger Live, announced taproot support in their
+  [v2.35.0 release][ledger v2.35.0] as an experimental feature.
+
+- **Muun wallet supports taproot:**
+  Muun wallet enabled taproot address support after activation occurred,
+  including the ability for users to default to taproot receive addresses.
+
+- **Lightning-based social feed squeaknode launches:**
+  [Squeaknode][squeaknode github] is a Twitter-like P2P network based on LN
+  (using LND). Users can create, view, buy, and sell messages known as "squeaks".
+
+- **Kollider launches alpha LN-based trading platform:**
+  Kollider's latest [announcement][kollider blog] details the derivative
+  platform's features including LN deposits and withdrawals, LNAUTH and LNURL
+  support, and a LN-based referral scheme.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [LND 0.14.0-beta.rc3][] is a release candidate that includes
+  additional [eclipse attack][topic eclipse attacks] protection (see
+  [Newsletter #164][news164 ping]), remote database support ([Newsletter
+  #157][news157 db]), faster pathfinding ([Newsletter #170][news170
+  path]), improvements for users of Lightning Pool ([Newsletter
+  #172][news172 pool]), and reusable [AMP][topic amp] invoices
+  ([Newsletter #173][news173 amp]) in addition to many other features
+  and bug fixes.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1
+repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][rust bitcoin repo], [BTCPay Server][btcpay server repo],
+[BDK][bdk repo], [Bitcoin Improvement Proposals (BIPs)][bips repo], and
+[Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #22934][] adds a verification step after both ECDSA signatures and
+  [schnorr signatures][topic schnorr signatures] are created.  This may
+  prevent the software from disclosing an incorrectly-generated
+  signature that may leak information about the private key or nonce
+  used to generate it.  This follows advice given in a previous update
+  to [BIP340][] (see [Newsletter #87][news87 bips886]) previously discussed in
+  [Newsletter #83][news83 safety].
+
+- [Bitcoin Core #23077][] enables address relay via CJDNS, making CJDNS
```

Comment:
Perhaps link to https://github.com/cjdelisle/cjdns for readers who aren't familiar with CJDNS.